code/game/gamemodes/miniantags/abduction/machinery/pad.dm

```diff
@@ -16,30 +16,27 @@
 	flick("alien-pad", src)
 	for(var/mob/living/target in loc)
 		target.forceMove(teleport_target)
-		spawn(0)
-			anim(target.loc,target,'icons/mob/mob.dmi',,"uncloak",,target.dir)
+		PoolOrNew(/obj/effect/overlay/temp/dir_setting/ninja, list(get_turf(target), target.dir))
```

Comment:
I dunno why you guys still insist on using Pooling when it *still* has a net loss in performance, overall, even under 511.